MAUI/Barcode/customization.md

```diff
@@ -0,0 +1,364 @@
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        barcode.TextStyle = new BarcodeTextStyle()
+        {
+            FontAttributes = FontAttributes.Italic,
+            FontFamily = "Time",
+            FontSize = 16,
+            TextColor = Colors.Red
+        };
+
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Customization](images/customization/maui-text-customization.png)
+
+### Text spacing
+
+The space between the text and the barcode can be controlled by the `TextSpacing` property of barcode generator. By default, the value of `TextSpacing` is 2.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+            <barcode:SfBarcodeGenerator Value="12634388927" 
+                                        HeightRequest="150"
+                                        WidthRequest="300" 
+                                        ShowText="True" 
+                                        TextSpacing="25">
+            </barcode:SfBarcodeGenerator>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 300;
+        barcode.Value = "12634388927";
+        barcode.ShowText = true;
+        barcode.TextSpacing = 25;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Spacing](images/customization/maui-text-spacing.png)
+
+### Horizontal text alignment
+
+The horizontal alignment of the text can be controlled by `TextAlignment` property of barcode generator. The displayed value can be positioned at `Start`, `Center` or `End` of the control. The default value of `TextAlignment` property is center.
+
+{% tabs %}
+
+{% highlight xaml hl_lines="5" %}
+
+       <barcode:SfBarcodeGenerator Value="12634" 
+                                   HeightRequest="150"
+                                   WidthRequest="240" 
+                                   ShowText="True" 
+                                   TextAlignment="End"/>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="6" %}
+
+        SfBarcodeGenerator barcode = new SfBarcodeGenerator();
+        barcode.HeightRequest = 150;
+        barcode.WidthRequest = 240;
+        barcode.Value = "12634";
+        barcode.ShowText = true;
+        barcode.TextAlignment = TextAlignment.End;
+        this.Content = barcode;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Barcode Generator Text Alignment](images/customization/maui-text-alignment.png)
+
+## Bar customization
+
+### Bar width customization
+
+Both the one dimensional and the two dimensional symbology support the `Module` property. The property is used to define the size of the smallest line or dot of the barcode.
```

Comment:
Both the one-dimensional and the two-dimensional symbology support the `Module` property. This property is used to define the size of the smallest line or dot of the barcode.

Reply:
Changes updated.